classic-editor.php

```diff
@@ -150,7 +150,7 @@ public static function remove_gutenberg_hooks( $remove = 'all' ) {
 		remove_action( 'submitpost_box', 'gutenberg_intercept_meta_box_render' );
 		remove_action( 'submitpage_box', 'gutenberg_intercept_meta_box_render' );
 		remove_action( 'edit_page_form', 'gutenberg_intercept_meta_box_render' );
-		remove_action( 'edit_form_advanced', 'gutenberg_intercept_meta_box_render' );

```

Comment:
Thinking we need to keep this for older Gutenberg versions on WP < 5.0.

Reply:
Can put that one back in if needed :)